src/Core/CSharpVB/RuleSetGenerator.cs

```diff
@@ -93,9 +94,12 @@ public RuleSet Generate(string language, IEnumerable<SonarQubeRule> activeRules,
         private static bool IsSupportedRuleRepo(IGrouping<string, SonarQubeRule> analyzerRules)
         {
             var partialRepoKey = analyzerRules.Key;
-            return !string.IsNullOrEmpty(partialRepoKey);
+            return !string.IsNullOrEmpty(partialRepoKey) && !IsExcludedRuleRepo(partialRepoKey);
         }
 
+        private static bool IsExcludedRuleRepo(string partialRepoKey) =>
+            partialRepoKey.IndexOf("sonaranalyzer.security", StringComparison.OrdinalIgnoreCase) >= 0;
+
```

Comment:
There were lots of ways I could have written this check. I went for a fairly loose check that would match VB rules if they were added.